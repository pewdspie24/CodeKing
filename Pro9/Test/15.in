10
1002504354838 1003449246071
1001933673734 1003841295461
1002183136440 1006297995795
1001403378110 1002900411946
1001141158899 1003693830155
1003853034375 1006565157811
1000561187649 1003777311888
1003384749548 1004366105654
1000073691032 1004131365091
1003127703052 1006163815749
