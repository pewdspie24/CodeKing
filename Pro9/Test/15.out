9917598
20413091
37557708
15666586
28019026
23078441
34287204
9739878
44059381
25931354
